code/queue.rb

```diff
@@ -398,15 +398,15 @@ def run_queue_script!(msg)
           $log.debug("set ENV, now executing #{script_path}")
 
           # bash -lc will execute the command but first re-initializing like a new login (reading .bashrc, etc.)
-          exec_prefix = @config.exec_prefix || "bash -lc "
+          exec_prefix = @config.exec_prefix || "bash -lc"
           if exec_prefix.empty?
```

Comment:
False, it can be empty if the config file presents an empty string:
```
$ irb
irb(main):001:0> foo = "" || "bar"
=> ""
irb(main):004:0> foo = nil || "bar"
=> "bar"
```

Since the config file is JSON, it can pass an empty string or a `null` explicitly (which would be weird, but it's doable).